INSTALL.md

```diff
@@ -1,143 +1,125 @@
-
-   - fast-histogram >= 0.10
-   - h5py
-   - iminuit >= 2
-   - line_profiler
-   - llvmlite
-   - matplotlib >= 3.0
-   - nlopt
-   - numba >= 0.53
-   - numpy >=1.17, < 1.23
-   - pandas
-   - pint <=0.19
-   - py-cpuinfo
-   - pyarrow
-   - scikit-learn <= 1.1.2
-   - scipy >= 1.6
-   - simplejson == 3.18.4
-   - sympy
-   - tables
-   - tabulate
-   - tqdm
-   - uncertainties
-   - kiwisolver >= 1.3.1
-   - cycler >= 0.10
-   - packaging >= 20.0
-   - fonttools >= 4.22.0
-   - python-dateutil >= 2.7
-   - pillow >= 8
-   - contourpy >= 1.0.1
-   - pyparsing >= 2.3.1
-   - threadpoolctl >= 2.0.0
-   - joblib >= 1.0.0
-   - numexpr
-   - pytz >= 2020.1
-   - tzdata >= 2022.1
-   - mpmath >= 0.19
-   - blosc2 >= 2.3.0
-   - future
-   - ndindex >= 1.4
-   - msgpack
-   - six >= 1.5
-2. Create conda environment/ virtual environment for the installation of PISA. Assume that the name of the environment is `pisa_env` you can choose your preferred version of python >= 3.10
+1. Install the essential dependencies
+   
+   Choose either Anaconda (~4.4 GB) OR Miniconda (~480 MB)
+
+   **Anaconda** (https://docs.anaconda.com/anaconda/), e.g.
+   ```bash
+   mkdir -p PATH_TO_ANACONDA/anaconda3
+   wget https://repo.anaconda.com/archive/Anaconda3-<INSTALLER_VERSION>-Linux-x86_64.sh -O PATH_TO_ANACONDA/anaconda3/anaconda.sh
+   bash PATH_TO_ANACONDA/anaconda3/anaconda.sh -b -u -p PATH_TO_ANACONDA/anaconda3
+   rm PATH_TO_ANACONDA/anaconda3/anaconda.sh
+   ```
+   You can view a list of available installer versions at https://repo.anaconda.com/archive/.
+   
+   **Miniconda** (https://docs.anaconda.com/miniconda/), e.g.
+   ```bash
+   mkdir -p PATH_TO_ANACONDA/miniconda3
+   wget https://repo.anaconda.com/miniconda/Miniconda3-<INSTALLER_VERSION>-Linux-x86_64.sh -O PATH_TO_ANACONDA/miniconda3/miniconda.sh
+   bash PATH_TO_ANACONDA/miniconda3/miniconda.sh -b -u -p PATH_TO_ANACONDA/miniconda3
+   rm PATH_TO_ANACONDA/miniconda3/miniconda.sh
+   ```
+   You can view a list of available installer versions at https://repo.anaconda.com/miniconda/.
+   
+   **If you install on the cobalts, rather use /data/user/YOURNAME/ than $HOME/ for PATH_TO_ANACONDA**
+   
+   **Note** that you will also need git. It is already installed on the cobalts, ask your local system administrator if it is not on your local machine. The other non-python requirements are listed [here](https://github.com/icecube/pisa/blob/master/INSTALL.md#required-dependencies) and should already come with the conda environment.
+
+   Other required libraries will be installed automatically during the setup (listed in https://github.com/icecube/pisa/blob/master/setup.py). If something is missing, you can install it via pip afterwards.
+
+2. Create an environment for the installation of PISA (after activating anaconda). You can choose your preferred version of python >= 3.10. E.g. for miniconda with python 3.10 use:
```

Comment:
This "preferred version of python >= 3.10" has somehow made it in here again, even though > isn't supported.

Reply:
See https://github.com/icecube/pisa/issues/779

Reply:
Right, ideally we shouldn't give a version number here at all because that means we have to update the file regularly. However, we should rather work on the "doesn't work with python>3.10" issue